descwl_shear_sims/shifts.py

```diff
@@ -129,3 +145,38 @@ def get_random_shifts(*, rng, dim, buff, size):
     shifts['dy'] = rng.uniform(low=low, high=high, size=size)
 
     return shifts
+
+
+def get_pair_shifts(*, rng, sep):
+    """
+    get a set of gridded shifts, with random shifts at the pixel scale
+
+    Parameters
+    ----------
+    rng: numpy.random.RandomState
+        The random number generator
+    sep: float
+        Separation of pair in arcsec
+
+    Returns
+    -------
+    shifts: array
+        Array with dx, dy offset fields for each point, in
+        arcsec
+    """
+
+    shifts = np.zeros(2, dtype=[('dx', 'f8'), ('dy', 'f8')])
+
+    angle = rng.uniform(low=0, high=2*np.pi)
+    shift_radius = sep / 2
+
+    dx = np.cos(angle)*shift_radius
+    dy = np.sin(angle)*shift_radius
+
+    shifts['dx'][0] = dx
+    shifts['dy'][0] = dy
+
+    shifts['dx'][1] = -dx
+    shifts['dy'][1] = -dy
```

Comment:
We should dither these positions?

Reply:
We have not been dithering the grid, relying on dithering at the image level.  I actually have an issue on this #155

Reply:
I think we want dithers in the coadd system too just be safe. It certainly can't hurt. I'd strongly suggest doing it.

Reply:
the shifts codes do not know about the pixel scale, so this would require some restructuring

Reply:
An easier option would be to require image dithers

Reply:
Image dithers still leave the objects at the same spot in the coadd system. This is still a potential source of bias.

Reply:
actually we are dithering the grid, so that issue must be out of date